packages/expo-router/src/link/preview/native/index.ios.ts

```diff
@@ -0,0 +1,4 @@
+export { default as LinkPreviewNativeView } from './LinkPreviewNativeView';
+export { default as LinkPreviewNativePreviewView } from './LinkPreviewNativePreviewView';
+export { default as LinkPreviewNativeTriggerView } from './LinkPreviewNativeTriggerView';
+export { default as LinkPreviewNativeActionView } from './LinkPreviewNativeActionView';
```

Comment:
Avoid barrel files as they complicate bundling and slow down overall bundle times.